dotcom-rendering/src/components/ProductCarouselCard.tsx

```diff
@@ -0,0 +1,145 @@
+import { css } from '@emotion/react';
+import {
+	headlineBold20,
+	headlineMedium17,
+	lifestyle,
+	space,
+	textSans17,
+	textSansBold15,
+	textSansBold17,
+} from '@guardian/source/foundations';
+import type { ArticleFormat } from '../lib/articleFormat';
+import type { ProductBlockElement } from '../types/content';
+import { ProductCardButtons } from './ProductCardButtons';
+import { ProductCardImage } from './ProductCardImage';
+
+export type ProductCarouselCardProps = {
+	product: ProductBlockElement;
+	format: ArticleFormat;
+	showReadMore?: boolean;
+};
+
+const baseCard = css`
+	display: flex;
+	width: 280px;
+	flex-direction: column;
+	align-items: flex-start;
+	padding: ${space[3]}px 10px ${space[4]}px 10px;
+`;
+
+const productCarouselCardHeading = css`
+	${headlineBold20};
+	color: ${lifestyle[300]};
+`;
+
+const brandAndProductName = css`
+	${headlineMedium17};
+`;
+
+const readMoreCta = css`
+	${textSansBold15};
+	text-decoration-line: underline;
+	text-decoration-color: #dcdcdc; /* stylelint-disable-line */
+	color: ${lifestyle[400]};
+	text-underline-offset: 20%;
+	padding-bottom: ${space[2]}px;
+`;
+
+const priceStyle = css`
+	${textSansBold17};
+	padding-top: ${space[1]}px;
+	padding-bottom: ${space[2]}px;
+`;
+
+const buttonWrapper = css`
+	grid-area: buttons;
+	display: flex;
+	flex-direction: column;
+	gap: ${space[1]}px;
+	justify-content: center;
+	align-items: center;
+	align-self: stretch;
+`;
+
+const imageArea = css`
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	align-self: stretch;
+	aspect-ratio: 1 / 1;
+	height: 280px;
+
+	img {
+		width: 100%;
+		height: 280px;
+		object-fit: cover;
+	}
+`;
+
+const brandAndProductNameRow = css`
+	display: block;
+	line-height: 1.3;
+`;
+
+const brandAndProductNameInline = css`
+	${headlineMedium17};
+	display: inline;
+`;
+
+const productNameStyle = css`
+	${textSans17};
+`;
+
+export const ProductCarouselCard = ({
+	product,
+	format,
+	showReadMore,
+}: ProductCarouselCardProps) => {
+	const hasHeading = !!product.primaryHeadingHtml;
+
+	const firstCta = product.productCtas[0];
+
+	return (
+		<div css={baseCard}>
+			{hasHeading && (
+				<>
+					<div css={productCarouselCardHeading}>
+						{product.primaryHeadingHtml}
+					</div>
```

Comment:
```suggestion
					<div
						css={productCarouselCardHeading}
						dangerouslySetInnerHTML={{
							__html: product.primaryHeadingHtml,
						}}
					/>
```

As this is html we can set it using dangerouslySetInnerHTML which means `<strong>` and `<em>` will be rendered correctly